test/test_cursor.py

```diff
@@ -225,11 +223,14 @@ def test_cursor_close(self, memgraph_server):
 
         cursor2.close()
 
-        assert cursor.fetchmany(10) == [(n, ) for n in range(1, 11)]
```

Comment:
I think this should work without any changes. :thinking:

Reply:
Yep, you right. It works, but since fetch is called twice, once you pull the last result, the cursor becomes immediately closable, and then the `InterfaceError` can't be tested anymore. This is changed to preserve the entire testing logic.